packages/nextcloud/lib/src/api/spreed.openapi.json

```diff
@@ -900,97 +900,139 @@
+                            "systemtag",
+                            "talk-attachment",
+                            "talk-poll",
+                            "user",
+                            "user-group",
+                            "group"
+                        ]
                     },
                     "id": {
                         "type": "string"
                     },
                     "name": {
                         "type": "string"
                     },
-                    "server": {
-                        "type": "string"
-                    },
                     "link": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The full URL to the file",
+                        "example": "http://localhost/index.php/apps/announcements/#23"
                     },
                     "call-type": {
                         "type": "string",
-                        "enum": [
-                            "one2one",
-                            "group",
-                            "public"
-                        ]
+                        "description": "The type of the call: one2one, group or public",
+                        "example": "one2one"
                     },
                     "icon-url": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The icon url to use as avatar",
+                        "example": "https://localhost/ocs/v2.php/apps/spreed/api/v1/room/R4nd0mToken/avatar"
                     },
                     "message-id": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The id of a message that was referred to",
+                        "example": "12345"
                     },
                     "boardname": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The display name of board which contains the card",
+                        "example": "Personal"
                     },
                     "stackname": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The display name of the stack which contains the card in the board",
+                        "example": "To do"
                     },
                     "size": {
-                        "oneOf": [
-                            {
-                                "type": "string"
-                            },
-                            {
-                                "type": "integer"
-                            }
-                        ]
+                        "type": "string",
+                        "description": "The file size in bytes",
+                        "example": "3145728"
                     },
```

Comment:
Why can't it be an integer anymore?
Is this an error or was it just false before?

Reply:
Ah yeah I forgot to patch that back in